specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.6/TextAnalytics.json

```diff
@@ -0,0 +1,3055 @@
+          "THPopulationIdentificationCode",
+          "TRNationalIdentificationNumber",
+          "UKDriversLicenseNumber",
+          "UKElectoralRollNumber",
+          "UKNationalHealthNumber",
+          "UKNationalInsuranceNumber",
+          "UKUniqueTaxpayerNumber",
+          "USUKPassportNumber",
+          "USBankAccountNumber",
+          "USDriversLicenseNumber",
+          "USIndividualTaxpayerIdentification",
+          "USSocialSecurityNumber",
+          "UAPassportNumberDomestic",
+          "UAPassportNumberInternational",
+          "Organization",
+          "Email",
+          "URL",
+          "Age",
+          "PhoneNumber",
+          "IPAddress",
+          "Date",
+          "Person",
+          "Address",
+          "All",
+          "Default"
+        ]
+      },
+      "type": "array",
+      "uniqueItems": true
+    },
+    "EntityResolutions": {
+      "properties": {
+        "dateTime.TimeRange": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeRangeResolution"
+          }
+        },
+        "dateTime.Time": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeResolution"
+          }
+        },
+        "quantity.Dimension": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DimensionResolution"
+          }
+        }
+      }
+    },
+    "TimeRangeResolution": {
+      "type": "object",
+      "properties": {
+        "timex": {
+          "type": "string",
+          "description": "timex expression representation"
+        },
+        "start": {
+          "type": "string",
+          "description": "beginning date/time of the interval"
+        },
+        "end": {
+          "type": "string",
+          "description": "ending date/time of the interval"
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
+          "description": "The probability of the resolution instance being correct for the extracted text."
+        }
+      }
+    },
+    "TimeResolution": {
+      "type": "object",
+      "properties": {
+        "timex": {
+          "type": "string",
+          "description": "timex expression representation"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "time"
+          ],
+          "description": "The resolution type"
+        },
+        "value": {
+          "type": "string",
+          "description": "The actual time that the extracted text denote."
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
+          "description": "probability of the resolution instance being correct for the extracted text."
+        }
+      }
+    },
+    "DimensionResolution": {
+      "type": "object",
+      "properties": {
+        "subtype": {
+          "type": "string",
+          "description": "Dimension subtype (e.g., speed, volume)"
```

Comment:
We may want to come up with a consistent use of nomenclature for `type` and `subtype` across all of the resolutions.  I see `type` used above in the `TimeResolution` object, which seems semantically pretty similar to `subtype` used here.  Could we maybe just pick one and use it across the board?

Reply:
That's a fair point. I think resolution type is reflected in the property name (e.g., "timeRange, "dimension", "time"). I can remove type inside each entity resolution and put it on the top level entity property. Then I can define it as an enum.

The subType can be renamed to "dimensionType" or something. I will work with the PM team as this also came from the Text Recognizers.